universal-login-react/src/core/services/DeleteAccountService.ts

```diff
@@ -0,0 +1,30 @@
+import {transactionDetails} from '../../core/constants/TransactionDetails';
+import {WalletService} from '@universal-login/sdk';
+
+interface ErrorsType {
+  usernameError: boolean;
+  verifyFieldError: boolean;
+}
+
+interface InputsType {
+  username: string;
+  verifyField: string;
+}
+
+const checkInputsAgainstError = (usernameLogged: string, inputs: InputsType) => ({
+  usernameError: inputs.username !== usernameLogged,
+  verifyFieldError: inputs.verifyField !== 'DELETE MY ACCOUNT'
+});
+
+export const getInputClassName = (inputError: boolean) => inputError ? 'delete-account-input-error' : 'delete-account-input';
+
+const doesAnyErrorExists = (errors: ErrorsType) => errors.usernameError || errors.verifyFieldError;
+
+export const deleteAccount = async (walletService: WalletService, inputs: InputsType, setErrors: (errors: ErrorsType) => void, onDeleteAccountClick: () => void) => {
```

Comment:
The `onDeleteAccountClick`, I'm not sure about the naming. Seems like it should be `onAccountDeleted`? I guess clicking 'delete account' happens before, which leds us here?

Reply:
👍